app/questionnaire/questionnaire_schema.py

```diff
@@ -135,10 +147,18 @@ def answer_dependencies(self) -> ImmutableDict[str, set[AnswerDependent]]:
         return ImmutableDict(self._answer_dependencies_map)
 
     @cached_property
-    # Type ignore: safe to assume _min_and_max_map return type
+    # Type ignore: make_immutable uses generic types so return type is manually specified, same for properties below.
     def min_and_max_map(self) -> ImmutableDict[str, ImmutableDict[str, int]]:
         return make_immutable(self._min_and_max_map)  # type: ignore
 
+    @cached_property
+    def lists_populated_by_list_collector(self) -> frozenset[str]:
+        return make_immutable(self._lists_populated_by_list_collector)  # type: ignore
+
+    @cached_property
+    def schema_dependent_lists(self) -> frozenset[str]:
+        return make_immutable(self._schema_dependent_lists)  # type: ignore
```

Comment:
Did you not bother with the type ignore comment since the first property explains it or was it missed?.